src/efisecdb.c

```diff
@@ -376,10 +383,7 @@ main(int argc, char *argv[])
 	INIT_LIST_HEAD(&infiles);
 	INIT_LIST_HEAD(&actions);
 
-	on_exit(free_actions, &actions);
-	on_exit(free_infiles, &infiles);
-	on_exit(maybe_free_secdb, &secdb);
-	on_exit(maybe_do_unlink, &outfile);
+	atexit(on_exit_cleanup);
```

Comment:
I expect this function is going away, but wanted to note that calling something on_exit_cleanup() and then passing it to atexit() is needlessly confusing.

Reply:
Agree that naming might be better. I find good naming hard.

Reply:
As do we all :)  There's no shame in code review finding things - if it didn't, it'd be a waste of time, after all.